src/main/kotlin/step2/application/parser/implement/SequentialParsingProcessor.kt

```diff
@@ -0,0 +1,34 @@
+package step2.application.parser.implement
+
+import step2.application.parser.ParsingProcessor
+import step2.domain.calculator.model.InputOperationCommand
+import step2.domain.operation.enums.Operator
+import java.lang.Integer.parseInt
+
+class SequentialParsingProcessor : ParsingProcessor {
+
+    companion object {
+        private const val SPLIT_DELIMITER = " "
+        private const val SPLIT_SIZE = 2
+    }
+
+    override fun proceed(string: String): List<InputOperationCommand> {
+        val splitStringArray = string.trim().split(SPLIT_DELIMITER)
+
+        check(splitStringArray.size.isOdd()) { "식의 원소 숫자는 홀수여야 합니다." }
+
+        val inputOperationCommandList = mutableListOf<InputOperationCommand>()
+
+        splitStringArray.chunked(size = SPLIT_SIZE) { chunk: List<String> ->
+            val number = parseInt(chunk.first())
```

Comment:
```suggestion
            val number = chunk.first().toInt()
```

코틀린이 제공하는 toInt() 확장함수를 사용하면 java 에 대한 의존(import)도 제거할 수 있습니다!

Reply:
사용할 때, 자바 메서드가 호출되서 이상했는데.. 확장함수로 사용하면 됐었군요 ㅋㅋㅋ [1e83e5b](https://github.com/next-step/kotlin-racingcar/pull/742/commits/1e83e5b7672cdb52efba9699cf9f0ba779076bf0)에서 변경했습니다.